lasagne/layers/conv.py

```diff
@@ -9,6 +9,7 @@
 
 
 __all__ = [
+    "conv_output_length",
```

Comment:
+1 for documenting it, but do we really want to export that as a first-class `lasagne.layers` citizen? I would have considered this to be an internal helper function to share code between convolution implementations.